backend/api/views.py

```diff
@@ -0,0 +1,64 @@
+from django.shortcuts import render
+
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from .models import User
+
+class LoginView(APIView):
+    
+    def post(self,request):
+
+        if 'user_name' in request.data:
+            user = User.objects.filter(username=request.data.get('user_name')).first()
+        else:
+            user = User.objects.filter(email=request.data.get('email')).first()
+
+        if not user or not user.check_password(request.data.get('password')):
```

Comment:
Returning token as a string of ok or not ok doesn't add much value. Same goes for message as well. I think we can remove token completely and return a message only when we need to return an error message.

Reply:
- Token depends on how we will handle users' session. 
- Error message should be sent when it is necessary